### DISCUSSION TITLE
reorderable is not working in table in v3

### DISCUSSION THREAD

**Discussion by liza-nt** [filamentphp/filament#8573] [Help]
i have position column and i want to reorder that but it is not working that same code working in v2 
```
 public static function table(Table $table): Table
    {
        return $table
            ->reorderable('position')

**Comment by leandrocfe** [comment]
any error? console error?
It is working here and [filament demo](https://demo.filamentphp.com/shop/brands?isTableReordering=true)
What is the Filament version?

**Reply by liza-nt** [reply]
oh i was using v3..i think there like more than 5000 data so that was not working ...then again it worked after failing few times
